internal/controller/constants/constants.go

```diff
@@ -65,6 +65,7 @@ const (
 	ModelRegistryModelVersionIdLabel     = "modelregistry.opendatahub.io/model-version-id"
 	ModelRegistryRegisteredModelIdLabel  = "modelregistry.opendatahub.io/registered-model-id"
 	ModelRegistryFinalizer               = "modelregistry.opendatahub.io/finalizer"
+	ModelRegistryServiceAnnotation       = "routing.opendatahub.io/external-address-rest"
```

Comment:
serviceAnnotation points to an external route?

Isn't better to rename this constant to something else that matches `external-address-rest`?

also, is `routing.opendatahub.io` required? Or can we keep the naming convention like the others?
`modelregistry.opendatahub.io/external-route`

Reply:
To give a bit of context, in each k8s service associated with a model registry instance, we annotate it with 
`routing.opendatahub.io/external-address-rest` (and also another one for grpc) with the model registry operator with the urls exposed by istio, this info is used by the dashboard and any other component that needs it to communicate with the model registry via http calls. Now, I think changing it introduces breaking changes with the clients, so if possible I would like to not do that, but yes I can rename the variable if it makes it easier to think about it

Reply:
okay, thanks for the explanation.